packages/samples/grpc-kiosk-example/types.cadl

```diff
@@ -26,7 +26,6 @@ model LatLng {
   longitude: float64;
 }
 
-@body
```

Comment:
why this change?

Reply:
the validation added on the body decorator only allows ModelProperty as the target, Maybe this is an incorrect assumption but I am not sure what you'd expect by putting @body on a model.